Wireframe/index.html

```diff
@@ -4,9 +4,66 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="/Wireframe/css/style.css">
   </head>
   <body>
```

Comment:
I'm not sure this meets the acceptance criteria. Where is the page title and short description shown on the wireframe?

Reply:
The HTML file and  The CSS file has different  folder path. first i was try just use  href ="style.css" but it was not work, then i used the folder name.